app/controllers/tokite/repositories_controller.rb

```diff
@@ -19,7 +22,7 @@ def new
     def create
       params[:names].each do |name|
         github_repo = octokit_client.repository(name)
-        Repository.hook!(octokit_client, github_repo)
+        Repository.hook!(octokit_client, github_repo) unless github_repo.archived
```

Comment:
It would be better if tokite shows error messages rather than ignoring archived repositories silently.
`flash` might help http://guides.rubyonrails.org/action_controller_overview.html#the-flash .

Reply:
Thanks! I changed to show error messages.